great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -290,13 +257,32 @@ def batch_request_options_to_batch_spec_kwarg_identifiers(
         return {self.column_name: options["quotient"]}
 
 
+class ColumnSplitterModInteger(_ColumnSplitter):
+    mod: int
+    method_name: Literal["split_on_mod_integer"] = "split_on_mod_integer"
+
+    @property
+    def param_names(self) -> List[str]:
+        return ["remainder"]
+
+    def splitter_method_kwargs(self) -> Dict[str, Any]:
+        return {"column_name": self.column_name, "mod": self.mod}
+
+    def batch_request_options_to_batch_spec_kwarg_identifiers(
+        self, options: BatchRequestOptions
+    ) -> Dict[str, Any]:
+        if "remainder" not in options:
+            raise ValueError(
+                "'remainder' must be specified in the batch request options to create a batch identifier"
```

Comment:
@billdirks I was under the impression that while "batch_identifier" is still an implementation detail (at least for the immediate term), this phrase is no longer present in the New Datasources vocabulary.  Thanks.

Reply:
Done.

Reply:
🙇